packages/conf-ppl/conf-ppl.1/opam

```diff
@@ -0,0 +1,17 @@
+opam-version: "1"
+author: "R. Bagnara, P. M. Hill, E. Zaffanella, A. Bagnara et. al."
+maintainer: "francois.berenger@inria.fr"
+homepage: "http://bugseng.com/products/ppl/"
+license: "GPL"
+build: [
+  ["sh" "-c" "cc test.c -lppl_c -lppl -lgmpxx -lgmp"]
+]
+depends: [
+  "conf-gmp"
+  "conf-mpfr"
```

Comment:
Can these dependencies be removed?  If gmp and mpfr are dependencies of ppl then it seems reasonable to leave the task of installing them to whatever/whoever installs ppl, ideally the system package manager.